src/pages/non-pass-finalisation/cat-home/__tests__/non-pass-finalisation.cat-home-test.page.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { ComponentFixture, async, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { IonicModule, NavController, Platform } from 'ionic-angular';
+import { NavControllerMock, PlatformMock } from 'ionic-mocks';
+import { AppModule } from '../../../../app/app.module';
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
+import { Store } from '@ngrx/store';
+import { StoreModel } from '../../../../shared/models/store.model';
+import { MockComponent } from 'ng-mocks';
+import { PracticeModeBanner } from '../../../../components/common/practice-mode-banner/practice-mode-banner';
+import { NonPassFinalisationCatHomeTestPage } from '../non-pass-finalisation.cat-home-test.page';
+import {
+  NonPassFinalisationViewDidEnter,
+  NonPassFinalisationValidationError,
+} from '../../non-pass-finalisation.actions';
+import { ActivityCodeComponent } from '../../../office/components/activity-code/activity-code';
+import { SetTestStatusWriteUp } from '../../../../modules/tests/test-status/test-status.actions';
+import * as testActions from '../../../../modules/tests/tests.actions';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { D255Component } from '../../../../components/test-finalisation/d255/d255';
+import { LanguagePreferencesComponent } from
+'../../../../components/test-finalisation/language-preference/language-preferences';
+import { DebriefWitnessedComponent } from
+'../../../../components/test-finalisation/debrief-witnessed/debrief-witnessed';
+import { FinalisationHeaderComponent } from
+'../../../../components/test-finalisation/finalisation-header/finalisation-header';
+import { D255Yes, D255No, DebriefWitnessed, DebriefUnwitnessed } from
+'../../../../modules/tests/test-summary/common/test-summary.actions';
+import { CandidateChoseToProceedWithTestInWelsh, CandidateChoseToProceedWithTestInEnglish } from
+'../../../../modules/tests/communication-preferences/communication-preferences.actions';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { configureTestSuite } from 'ng-bullet';
+
+describe('NonPassFinalisationCatHomeTestPage', () => {
+  let fixture: ComponentFixture<NonPassFinalisationCatHomeTestPage>;
+  let component: NonPassFinalisationCatHomeTestPage;
+  let store$: Store<StoreModel>;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        NonPassFinalisationCatHomeTestPage,
+        MockComponent(PracticeModeBanner),
```

Comment:
practice mode banner not needed

Reply:
Nice spot, that's most likely been missed multiple times, Will clean it up here and put up another PR to clean it up elsewhere